sched/sched/sched_removereadytorun.c

```diff
@@ -209,20 +209,16 @@ bool nxsched_remove_readytorun(FAR struct tcb_s *rtcb)
 
       if (rtrtcb != NULL && rtrtcb->sched_priority >= nxttcb->sched_priority)
         {
-          FAR struct tcb_s *tmptcb;
-
           /* The TCB at the head of the ready to run list has the higher
            * priority.  Remove that task from the head of the g_readytorun
            * list and add to the head of the g_assignedtasks[cpu] list.
```

Comment:
I think we need to modify these comments as well.
Please change the comments and merge them into this commit.